experiment_design_doc.md

```diff
@@ -415,12 +415,15 @@ The client side is tested using [Bats](https://github.com/bats-core/bats-core).
 ### Creating a new host
 
 1. Clone the repository locally
-2. Update the Makefile in the local repo with the intended `HOST` and `HOST_DIR`,
+2. Update the Makefile in the local repo with the intended `HOST` and `HOST_DIR`.
+3. Update `SERVER_HOST` in `client_side/.infrastructure/setup.sh` with the new host.
+4. Update `client_side/README.txt` with the new host.
```

Comment:
I'm wondering if we could do something about this step so we don't have to manually change so many files... Thoughts?

Reply:
That's a good idea. The only robust and scalable solution I see is to generate these files with the correct values when we change the host, similar to how static website generators work.

Regarding workarounds, we could either import these variables in the scripts (difficult for README.txt) or have a script that replaces the values.

Reply:
Yeah. That seems like a QOL thing, though. So I'll leave it to your discretion. If you feel it's worth pursuing you can open an issue to track it.